File: Furthest-Point-From-Origin.java
1class Solution {
2    public int furthestDistanceFromOrigin(String moves) {
3        int left = 0,right = 0, steps = 0;
4        for(char ch : moves.toCharArray()){
5            if(ch=='L') left++;
6            else if(ch=='R') right++;
7            else steps++;
8        }
9        return Math.abs(left-right)+steps;
10    }
11}

File: Furthest-Point-From-Origin.cpp
1class Solution {
2public:
3    int furthestDistanceFromOrigin(string moves) {
4        int L = 0, R = 0, dash = 0;
5        for(char ch : moves){
6            if(ch=='L') L++;
7            else if(ch=='R') R++;
8            else dash++;
9        }
10        int pos = abs(L-R);
11        return pos+dash; 
12    }
13};